python/cudf_polars/cudf_polars/dsl/ir.py

```diff
@@ -1291,6 +1291,42 @@ def __init__(
         self.children = ()
         self._id_for_hash = random.randint(0, 2**64 - 1)
 
+    @staticmethod
+    def _reconstruct(
+        schema: Schema,
+        pl_df: pl.DataFrame,
+        projection: Sequence[str] | None,
+        id_for_hash: int,
+    ) -> DataFrameScan:  # pragma: no cover
```

Comment:
Do we need this pragma? At least currently, all the tests in `tests/experimental` should be run and so I'd expect this to be covered.